### ISSUE TITLE
Foreign key on referenced entity in ng-admin

### ISSUE THREAD

**Issue by dumaron** [marmelab/ng-admin#940]
I'm trying to build an admin interface using ng-admin.
I'm using a `reference` field to link one main entity (Post, for example) to a different entity (Comment) that has a many-to-one relationship (one Post has many Comments).

On a `listView()` of Posts I must display the last Comment of each Post, but as I can see from the [relationships documentation of ng-admin](http://ng-admin-book.marmelab.com/doc/Relationships.html) I can't because in my API the foreign key is not on the main entity but on the secondary one.

In other words, I have `post_id` in the Comment entity but I am doing a `listView()` of Posts so I can't access any Comment.

Is there a workaround for this issue? Can I reference to an external entity from an entity that does not contain the reference key itself? 

I'll add a simple JSON to make the example more clear.

```
// Post
{ 
  id: 1,
  title: 'post n1',
  body: 'bla bla bla very cool'
}

// Comment
{ 
  id: 1,
  body: 'yo bro'
  post_id: 1
}
```

Thank you

**Comment by tdeheurles** [comment]
the [`referenced_list`](http://ng-admin-book.marmelab.com/doc/reference/Field.html#-referenced-list-field-type) is done to match that as I understand it.

```
nga.field("Comments", "referenced_list")
   .targetEntity(comments)
   .targetReferenceField("post_id")
   .targetField(nga.field("body"))
```

`reference` is when you have the `id` and want a field in the table where this id is the PrimaryKey

`reference_list` is when your primaryKey is in the other element

But I'm not sure that you can then get the first one and show this in `listView`.

**Comment by Kmaschta** [comment]
Hello,

As explained in the [README](https://github.com/marmelab/ng-admin/blob/master/README.md), the GitHub issue tracker is for bugs and enhancement requests.
So I close this issue, but feel free to write a question on [StackOverflow with the tag `ng-admin`](http://stackoverflow.com/questions/tagged/ng-admin).

Thanks for using ng-admin!
